### PR TITLE
feat: show stripe invoices

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/8480

Demo: https://www.loom.com/share/3c9a1cd4f47543869e7abc9f727f25a4

Note: this only works for invoices created after this PR. If that's problematic, we could fix this by fetching the `invoice.hosted_invoice_url` from Stripe in the `invoice.payUrl` type.

### To test

- [ ] Run `tripe listen --forward-to localhost:3000/stripe` in your terminal and upgrade to the Team tier
- [ ] Click on the paid invoice and see that it takes you to Stripe's UI
- [ ] See that you can download the invoice successfully

### CODE REVIEW

**Review by igorlesnenko — APPROVED** [review]
This looks good to me, but few things I noticed:

In the stripe invoice there is an email: `admin@parabol.co` while in our own invoices `love@parabol.co` and  `billing@parabol.co` should we update email in stripe? 

When I open some old invoice, it first opens in the old UI, but when I go back and click on it again it opens in stripe. Is it expected behaviour?

Once I upgraded an org, and got this error in logs
```
Error: This object cannot be accessed right now because another API request or Stripe process is currently accessing it. If you see this error intermittently, retry the request. If you see this error frequently and are making multiple concurrent requests to a single object, make your requests serially or at a lower rate.
```
After that I only able to see the estimate invoice, not paid:
image

**Comment by nickoferrall** [comment]
> should we update email in stripe?

Good eye! I don't think so as I believe they're all sent to Hubspot 

> When I open some old invoice, it first opens in the old UI, but when I go back and click on it again it opens in stripe

I think that's a caching issue, so it shouldn't be a problem

> Once I upgraded an org, and got this error in logs

Thanks for sharing! I've been getting that too. I created a PR to fix it here: https://github.com/ParabolInc/parabol/pull/8545

File: packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx [hunk 1]
```diff
@@ -90,9 +92,19 @@ const InvoiceRow = (props: Props) => {
...
+      window.open(payUrl, '_blank', 'noreferrer')
+    } else {
+      history.push(`/invoice/${invoiceId}`)
+    }
+  }
+
   return (
     <Row>
-      <RowLink rel='noopener noreferrer' target='_blank' to={`/invoice/${invoiceId}`}>
+      <RowLink onClick={goToInvoice}>
```
Comment (Dschoordsch) [inline packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx:107]: -1 Previously also internal invoices would open in a new window. I think we could just do a
```ts
      <RowLink rel='noopener noreferrer' target='_blank' to={payUrl || `/invoice/${invoiceId}`}>
```
here

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx [hunk 2]
```diff
@@ -43,18 +43,17 @@ const InfoRowRight = styled('div')({
...
   justifyContent: 'space-between',
   textDecoration: 'none',
-  width: '100%',
-  '&:hover': {
-    cursor: 'pointer'
-  }
+  width: '100%'
 })
 
+const RowLink = LinkStyles.withComponent(Link)
```
Comment (Dschoordsch) [inline packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx]: -1 Does not work, it redirects to
`http://localhost:3000/me/organizations/oU3Kik4zU4/https://invoice.stripe.com/...` for me

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx (hunk 2 above)
Comment (nickoferrall) [reply packages/client/modules/userDashboard/components/InvoiceRow/InvoiceRow.tsx]: 🤦‍♂️ I've replaced the RowLink element with an achor element using tailwind. It's fixed for me now!

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**